From ffbfd6d5d19a53df45b1190ec3225223178efac1 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 21 Sep 2022 15:40:38 +0530
Subject: [PATCH] Use private drm_hwcomposer instead of external drm_hwcomposer

Rename lib names to libxxx_orig, so that we use
vendor/intel/external/drm-hwcomposer instead of
external drm_hwcomposer.

Change-Id: I3e2fa1a1cc883c6e3f53907666295db71abb6115
Signed-off-by: svenate <salini.venate@intel.com>
---
 Android.bp       | 22 +++++++++++-----------
 tests/Android.bp |  6 +++---
 2 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/Android.bp b/Android.bp
index c9c94af..5ccb2d0 100644
--- a/Android.bp
+++ b/Android.bp
@@ -33,7 +33,7 @@ license {
 }
 
 cc_library_static {
-    name: "libdrmhwc_utils",
+    name: "libdrmhwc_utils_orig",
 
     srcs: ["utils/Worker.cpp"],
 
@@ -55,7 +55,7 @@ cc_library_static {
 // hwcomposer.drm.so
 // =====================
 cc_defaults {
-    name: "hwcomposer.drm_defaults",
+    name: "hwcomposer.drm_defaults_orig",
 
     shared_libs: [
         "libcutils",
@@ -73,7 +73,7 @@ cc_defaults {
         "external/drm_hwcomposer/include",
     ],
 
-    static_libs: ["libdrmhwc_utils"],
+    static_libs: ["libdrmhwc_utils_orig"],
 
     cflags: [
         "-Wall",
@@ -97,7 +97,7 @@ cc_defaults {
 }
 
 filegroup {
-    name: "drm_hwcomposer_common",
+    name: "drm_hwcomposer_common_orig",
     srcs: [
         "bufferinfo/BufferInfoGetter.cpp",
         "bufferinfo/BufferInfoMapperMetadata.cpp",
@@ -134,13 +134,13 @@ filegroup {
 
 // Kept only for compatibility with older Android version. Please do not use!
 cc_library_static {
-    name: "drm_hwcomposer",
+    name: "drm_hwcomposer_orig",
     defaults: ["hwcomposer.drm_defaults"],
     srcs: [":drm_hwcomposer_common"],
 }
 
 cc_library_shared {
-    name: "hwcomposer.drm",
+    name: "hwcomposer.drm_orig",
     defaults: ["hwcomposer.drm_defaults"],
     srcs: [
         ":drm_hwcomposer_common",
@@ -150,7 +150,7 @@ cc_library_shared {
 }
 
 cc_library_shared {
-    name: "hwcomposer.drm_minigbm",
+    name: "hwcomposer.drm_minigbm_orig",
     defaults: ["hwcomposer.drm_defaults"],
     srcs: [
         ":drm_hwcomposer_common",
@@ -160,24 +160,24 @@ cc_library_shared {
 
 // Used by hwcomposer.drm_imagination
 filegroup {
-    name: "drm_hwcomposer_platformimagination",
+    name: "drm_hwcomposer_platformimagination_orig",
     srcs: ["bufferinfo/legacy/BufferInfoImagination.cpp"],
 }
 
 // Used by hwcomposer.drm_hikey and hwcomposer.drm_hikey960
 filegroup {
-    name: "drm_hwcomposer_platformhisi",
+    name: "drm_hwcomposer_platformhisi_orig",
     srcs: ["bufferinfo/legacy/BufferInfoMaliHisi.cpp"],
 }
 
 // Used by hwcomposer.drm_meson
 filegroup {
-    name: "drm_hwcomposer_platformmeson",
+    name: "drm_hwcomposer_platformmeson_orig",
     srcs: ["bufferinfo/legacy/BufferInfoMaliMeson.cpp"],
 }
 
 // Used by hwcomposer.drm_mediatek
 filegroup {
-    name: "drm_hwcomposer_platformmediatek",
+    name: "drm_hwcomposer_platformmediatek_orig",
     srcs: ["bufferinfo/legacy/BufferInfoMaliMediatek.cpp"],
 }
diff --git a/tests/Android.bp b/tests/Android.bp
index b25342e..5f4f119 100644
--- a/tests/Android.bp
+++ b/tests/Android.bp
@@ -1,5 +1,5 @@
 cc_library_shared {
-    name: "hwcomposer.filegroups_build_test",
+    name: "hwcomposer.filegroups_build_test_orig",
     defaults: ["hwcomposer.drm_defaults"],
 
     srcs: [
@@ -29,7 +29,7 @@ package {
 }
 
 cc_test {
-    name: "hwc-drm-tests",
+    name: "hwc-drm-tests_orig",
 
     srcs: ["worker_test.cpp"],
 
@@ -45,7 +45,7 @@ cc_test {
 
 // Tool for listening and dumping uevents
 cc_test {
-    name: "hwc-drm-uevent-print",
+    name: "hwc-drm-uevent-print_orig",
 
     srcs: ["uevent_print.cpp"],
 
-- 
2.33.0

